dcist-master/Dockerfile

```diff
@@ -169,24 +170,6 @@ RUN cd ~/cpp_deps \
   && cmake -B build \
   && sudo cmake --build build --config Release --target install
 
-# Build Ian's ORBSLAM3 version with a few patches (thanks Varun)
```

Comment:
Can we keep this commented? We will  need to deal with orbslam in the future so it may be good to have traceability.

Reply:
Yeah sure thing. I'll add in the commented block